src/components/Api.js

```diff
@@ -0,0 +1,133 @@
+export default class Api{
+    constructor(options, loadUserInfoCallback, loadCardsCallback, editProfileCallback, addNewCardCallback){
```

Comment:
` loadUserInfoCallback, loadCardsCallback, editProfileCallback, addNewCardCallback` are not needed in `Api`. You need to do that in `index.js` where you call methods